generators/app/templates/tsconfig.json

```diff
@@ -12,7 +12,7 @@
         "rootDir": "./src",
         "outDir": "./dist",
         "baseUrl": "./",
-        "moduleResolution": "Node",
+        "moduleResolution": "node",
```

Comment:
I checked this also and `Node` was also fine - anyways - that's better like this...